src/clr-angular/utils/a11y/aria-live.service.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { DOCUMENT } from '@angular/common';
+import { Inject, Injectable, NgZone, OnDestroy } from '@angular/core';
+import { uniqueIdFactory } from '../id-generator/id-generator.service';
+
+export type AriaLivePoliteness = 'off' | 'polite' | 'assertive';
```

Comment:
A Enum could make this a little cleaner so the referencing/consuming code does not have to introduce additional string usage.

Reply:
I like the idea 👍